test/test_core.py

```diff
@@ -2804,23 +2801,20 @@ def test_pthread_thread_local_storage(self):
 
   @node_pthreads
   def test_pthread_cleanup(self):
-    self.set_setting('EXIT_RUNTIME')
     self.set_setting('PTHREAD_POOL_SIZE', 4)
     self.do_run_in_out_file_test('pthread/test_pthread_cleanup.cpp')
 
   @node_pthreads
   def test_pthread_setspecific_mainthread(self):
-    self.set_setting('EXIT_RUNTIME')
-    print('.. return')
+    logger.debug('.. return')
     self.do_runf(test_file('pthread/test_pthread_setspecific_mainthread.c'), 'done!', emcc_args=['-DRETURN'])
-    print('.. exit')
+    logger.debug('.. exit')
     self.do_runf(test_file('pthread/test_pthread_setspecific_mainthread.c'), 'done!', emcc_args=['-DEXIT'])
-    print('.. pthread_exit')
+    logger.debug('.. pthread_exit')
```

Comment:
Why not keep these are print statements?  Our tests tend to use printing to stdout quite a lot.. if we want to avoid that we should do it in a universal way I think.

Reply:
Yeah I just saw that some tests already use `logger.debug` so I figured these are likely old-style leftovers. Since they annoyed me a bit whenever I ran `./test/runner core.*thread*`, I thought no harm in converting them too.

Reply:
Do you want me to revert this one?

Reply:
Yeah I think revert this since its unrelated.

Reply:
Done.